app/pom.xml

```diff
@@ -71,6 +84,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.keycloak</groupId>
+      <artifactId>keycloak-authz-client</artifactId>
+      <version>2.5.7.Final-redhat-2</version>
```

Comment:
Maybe not a good idea for the community variant of the booster?

Reply:
Don't know. There is another request to baseline on this version. Let's take it offline

Reply:
Should be 2.5.5.Final for community